src/plugins/condition/utils/styleUtils.js

```diff
@@ -77,11 +77,24 @@ const aggregateStyleValues = (accumulator, currentStyle) => {
         }
     });
     return accumulator;
-};
+}
+
+function getStaticStyleForItem(domainObject, id) {
```

Comment:
Confused as to why this PR includes a new function?

Reply:
It doesn't. Changed it from an existing function expression.

Reply:
Verified code was moved around. This function is not new.